src/main/java/frc/robot/subsystems/ClawSubsystem.java

```diff
@@ -14,8 +14,10 @@ public class ClawSubsystem extends SubsystemBase {
   /** Creates a new ClawSubsystem. */
   public ClawSubsystem() {}
 
-  private final CANSparkMax motleft = new CANSparkMax(14, MotorType.kBrushless);
-  private final CANSparkMax motright = new CANSparkMax(13, MotorType.kBrushed);
+  private final CANSparkMax motleft = new CANSparkMax(5, MotorType.kBrushless);
+
+  private final CANSparkMax motright = new CANSparkMax(1, MotorType.kBrushless);
```

Comment:
Were these can id's supposed to change in addition to the motor type?

Reply:
does it matter for right now since we dont know the ids on the new robot